solidity/ecdsa/test/WalletRegistry.WalletCreation.test.ts

```diff
@@ -2134,6 +2144,18 @@ describe("WalletRegistry - Wallet Creation", async () => {
             it("should unlock the sortition pool", async () => {
               await expect(await sortitionPool.isLocked()).to.be.false
             })
+
+            it("should refund ETH to a submitter", async () => {
+              const postDkgApprovalSubmitterBalance = await provider.getBalance(
+                await submitter.getAddress()
+              )
+              const diff = postDkgApprovalSubmitterBalance.sub(submitterBalance)
+
+              expect(diff).to.be.gt(0)
+              expect(diff).to.be.lt(
+                ethers.utils.parseUnits("40000", "gwei") // 0,00004 ETH
```

Comment:
This reply addresses the comment above and a couple of comments below because they are related.

Yes, the default gas price for hardhat is 1 gwei.

When running tests locally for gas prices 10, 50, 200 & 500 gwei it is very tricky to pick one range that fits all the testing scenarios. By looking at the [history chart](https://ycharts.com/indicators/ethereum_average_gas_price) we see that most of the time the gas < 200gwei. Today Feb 18 it's 77 gwei. There were spikes > 200 gwei but they didn't last long. I tried to adjust the numbers so that the refunds will be reasonably net-positive (most of the scenarios) or in some instances a bit net-negative. 

Current gas consumption and ETH refund precision in tests work fine when the gas price is assumed to be between 10-200 gwei. Even if the gas price is 200 gwei the biggest net-positive precision is 0.002ETH which today gives $5.59. The biggest net-negative precision for the gas price 200 gwei is 0.001ETH which today gives $2.79 In reality they are a bit smaller, so the tests have some slack. We can eliminate a net-negative, but at the cost of net-positive, ie. the net-positive will be higher than 0.002ETH in certain cases.

If the gas price jumps to > 200 gwei and the net-positve outcome won't be acceptable anymore the DAO will need to adjust the `dkgApprovalGasOffset` accordingly, ie. lower it. 

`dkgResultSubmissionGas` is set to 300k, because most of the tests showed gas usage around 300k (checked programmatically, not in hardhat's gas result)

`dkgApprovalGasOffset` is set to 65k in try&error approach. Dkg result param is about `70k` depending on the testing scenario. The rest of the gas for `approveDkgResult` was calculated on the fly.

commit: [49d10aa](https://github.com/keep-network/keep-core/pull/2856/commits/49d10aa7695039cf4deff587a7b265fe0cc7cbbd)